x/feegrant/keeper/keeper_test.go

```diff
@@ -172,14 +173,16 @@ func (suite *KeeperTestSuite) TestUseGrantedFee() {
 	// some helpers
 	atom := sdk.NewCoins(sdk.NewInt64Coin("atom", 555))
 	eth := sdk.NewCoins(sdk.NewInt64Coin("eth", 123))
+
+	exp := ctx.BlockTime().AddDate(1, 0, 0)
 	future := &types.BasicFeeAllowance{
 		SpendLimit: atom,
-		Expiration: types.ExpiresAtHeight(5678),
+		Expiration: &exp,
 	}
 
 	expired := &types.BasicFeeAllowance{
 		SpendLimit: eth,
-		Expiration: types.ExpiresAtHeight(55),
+		Expiration: &exp,
```

Comment:
I'm a bit confused on this line - does this actually trigger expiry?

Reply:
[comment](https://github.com/cosmos/cosmos-sdk/pull/9206#discussion_r622786624)